lib/Browser.js

```diff
@@ -82,8 +82,11 @@ Browser.prototype.start = function () {
       if (matches.length === 0) return
 
       matches.forEach(function (service) {
-        if (self._serviceMap[service.fqdn]) return // ignore already registered services
-        self._addService(service)
+        if (self._serviceMap[service.fqdn]) {
+          self._updateService(service) // ignore already registered services
```

Comment:
Would suggest to either remove or update the code comment here to reflect updated functionality 👍 

```suggestion
          self._updateService(service)
```

Reply:
Ooops, fixed